src/NUnitFramework/framework/TestContext.cs

```diff
@@ -75,6 +76,18 @@ public static TextWriter Out
             get { return TestExecutionContext.CurrentContext.OutWriter; }
         }
 
+#if !NETCF && !SILVERLIGHT && !PORTABLE
+        /// <summary>
+        /// Gets a TextWriter that will send output directly to Console.Error
+        /// </summary>
+        public static TextWriter Error = new EventListenerTextWriter("Error", Console.Error);
+
+        /// <summary>
+        /// Gets a TextWriter for use in displaying immediate progress messages
+        /// </summary>
+        public static readonly TextWriter Progress = new EventListenerTextWriter("Progress", Console.Error);
```

Comment:
Double check my understanding, Progress goes to Console.Error because we are capturing Console.Out, but in this case it will not get output in red, correct? I will probably answer my question as I get further down in the code review :smile: